interview-questions.md

```diff
@@ -10,39 +10,39 @@ Type your initials here to confirm you read the instructions:
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statement use if/else statements different operators to get an output based on met conditions.
 
-Researched answer:
+Researched answer: Decision structures, also called decision trees, conditional statements, or if/else statements, are fundamental to computer programming. Conditional statements are a sequence of well-defined instructions that produce a unique output based on the value of the input. Conditionals follow a flowchart-like structure and allow for logic within the code. -Learn Academy Syllabus, JavaScript Conditionals
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a system that allow developers to work together on different projects by allowing them to create local repositories to maintain version control.
 
-Researched answer:
+Researched answer: Git is a distributed version control system, which means that a local clone of the project is a complete version control repository. These fully functional local repositories make it easy to work offline or remotely. Developers commit their work locally, and then sync their copy of the repository with the copy on the server. This paradigm differs from centralized version control where clients must synchronize code with a server before creating new versions of code. -https://learn.microsoft.com/en-us/devops/develop/git/what-is-git
```

Comment:
Make sure to read the instructions. Researched answers are not to be copy and pasted. They should be in your own words. Moving forward, I'll be kicking the assessment back to redo these, but I'll let it go for this first one.